apps/web/src/components/MissingSkills/MissingSkills.test.tsx

```diff
@@ -114,22 +152,30 @@ describe("MissingSkills", () => {
           experienceSkillRecord: { details: null },
         },
         {
-          ...defaultProps.requiredSkills[0],
-          experienceSkillRecord: { details: "" },
+          ...defaultProps.requiredSkills[1],
+          experienceSkillRecord: { details: "details" },
```

Comment:
Oh yeah, done as I thought empty string wasn't a good test choice as it is technically falsy